game-core/src/main/java/games/strategy/triplea/ui/DicePanel.java

```diff
@@ -69,7 +70,9 @@ private JLabel makeDiceRolledLabel(final List<Die> dice, final int value) {
   }
 
   private static String colorizeHitString(final Object hitsString) {
-    return "<font color='#8B0000'>" + hitsString + "</font>";
+    // On a dark theme, use red. Use a darker red with a light theme.
+    final String color = LookAndFeel.isCurrentLookAndFeelDark() ? "red" : "#8B0000";
```

Comment:
minor, any objections to extracting "#8B0000" to a named constant? EG: "DARKER_RED"

Reply:
Done.